instrumentation-security/cassandra-datastax-3/src/main/java/com/datastax/driver/core/SimpleStatement_Instrumentation.java

```diff
@@ -0,0 +1,63 @@
+package com.datastax.driver.core;
+
+import com.datastax.driver.core.querybuilder.CassandraUtils;
+import com.newrelic.api.agent.security.NewRelicSecurity;
+import com.newrelic.api.agent.security.schema.VulnerabilityCaseType;
+import com.newrelic.api.agent.security.schema.operation.SQLOperation;
+import com.newrelic.api.agent.weaver.MatchType;
+import com.newrelic.api.agent.weaver.Weave;
+import com.newrelic.api.agent.weaver.WeaveAllConstructors;
+import com.newrelic.api.agent.weaver.Weaver;
+
+import java.nio.ByteBuffer;
+import java.util.HashMap;
+import java.util.Map;
+
+@Weave(type= MatchType.ExactClass, originalName = "com.datastax.driver.core.SimpleStatement")
+public abstract class SimpleStatement_Instrumentation {
+    private final String query = Weaver.callOriginal();
+    private final Object[] values = Weaver.callOriginal();
+    private final Map<String, Object> namedValues = Weaver.callOriginal();
+    @WeaveAllConstructors
+    public SimpleStatement_Instrumentation() {
```

Comment:
Please weave all constructors separately and do not use weaving of member variables

Reply:
Fixed in [commit](https://github.com/newrelic/csec-java-agent/pull/70/commits/83d2181cbef1e2ea56fc5d4dd1aa429d0afd51b1)